src/App.tsx

```diff
@@ -124,12 +129,25 @@ function App() {
     }
   };
 
+  const setExtensionIdToLocalStorage = useCallback(
+    (_extensionId: string): void => {
+      const walletList = Object.values(walletMapEntries);
+      if (
+        walletList.some((entry) => entry.params.extensionId === _extensionId)
```

Comment:
nit. what are your thoughts on setting the result of this to some variable like `existsExtensionId` and then putting that variable in the if statement instead for readability?

Reply:
resolved in [de81e](https://github.com/MetaMask/test-dapp-multichain/pull/22/commits/de81e7e392022e41ac8edd4066a44826540fb4d5)